set.mm

```diff
@@ -677950,6 +677950,84 @@ collection and union ( ~ mnuop3d ), from which closure under pairing
+  jca ralbidv cbvalvw nfv nfa1 nfan elpwi sp ssin biimpi ex adantr simp3 simpl2
+  simprr simprl 3jca eximdv 3anass bitr4i mpbiran imim12d ralimdv imbi1i impexp
+  mpd albii dfss2 df-ral syl6ibr anim12d reximdv 3impia 3com23 syl3an2 ralrimia
+  3expa sylan2 impbid1 ralbiia ) AIZJZEBIZKZLZUUGFIZMZKZUULUUIJJZKZMZLZNZBEOZFE
+  JZPZUUHELZUUHUUILZGCUAZCIZUULQZNZCEOZGDUAZDIZMUUILZNZDUULOZRZGUUGPZNZBEOZFSZN
+  ZAEUUGEQZUVBUVTUVBUVTRUWAUVBUVCUVSUVBUUKBEOZFUVAPZUVCBEOZUVCUUTUWBFUVAUUSUUKB
+  EUUKUURUBUCUDUWCUWBUWDUWCUWBRUEUWBUWBFUFUVAUFUVAQUEEUGUHUEUULUFUINUWBUJUKULUU
+  KUVCBEUUKUUJELUVCEUUIUMUUHUUJEUNUOUCUPUWDUVCBTUVCUVCBEUQUVCBURUPUSUVBUVDUVEUV
+  FHIZQZNZCEOZUVMDUWEOZRZGUUGPZNZBEOZHSUVSUVBUUKUUGEUWEKZMZKZUWNUUOKZMZLZNZBEOZ
+  UWMHUVBUXAHUWNUVAQZUVBUXARUXBUWNELEUWEUMUWNEUWEEHVLUTVAVBUUTUXAFUWNUVAUULUWNU
+  IZUUSUWTBEUXCUURUWSUUKUXCUUNUWPUUQUWRUXCUUMUWOUUGUULUWNVCVDUXCUUPUWQUULUWNUUO
+  VEVFVGVHVIVJVKVMUWTUWLBEUUKUVDUWSUWKUUKUUJUUILUVDEUUIVNUUHUUJUUIUNUOUWSUWJGUU
+  GUWHGIZUWOQZUWSUXDUUGQZNUWIUVFEQZUWGNZCTUVEUVFUWNQZNZCTUWHUXEUXHUXJCUXHUVEUXG
+  UWFNZNUXJUXGUVEUWFVOUXKUXIUVEUXIUXKUVFEUWEVPVQVRVSVTUWGCEWECUXDUWNWAWBUWSUXFU
+  XEUWIUWSUXFUXEWCZUVJUVKUWQQZNZDTZUWIUXLUXDUWRQUXOUXLUWPUWRUXDUWSUXFUXEWDUXFUX
+  EUXDUWPQZUWSUXPUXFUXENUXDUUGUWOVPWFWGWHDUXDUWQWAWIUXOUVKUWEQZUVMNZDTUWIUXNUXR
+  DUXNUVJUXQUVLNZNUXRUXMUXSUVJUXMUXQUVLUXMEUWEUVKUVKUWNUUOWJWKUXMUVKUUOQZUVLUVK
+  UWNUUOWLUXTUVKWMQZUVLUVKUUIWNZWOUPXGWPUVJUXQUVLVOWIWQUVMDUWEWEWRUPWSWTXAXBUCX
+  CUVRUWMFHUULUWEUIZUVQUWLBEUYCUVPUWKUVDUYCUVOUWJGUUGUYCUVIUWHUVNUWIUYCUVHUWGCE
+  UYCUVGUWFUVEFHCXDVHVIUVMDUULUWEXEXFXHVHVIXIWRXGUHUVTUUTFUVAUVCUVSFUVCFXJUVRFX
+  KXLUULUVAQUVTUULELZUUTUULEXMUVCUVSUYDUUTUVSUVCUVRUYDUUTUVRFXNUVCUYDUVRUUTUVCU
+  YDUVRUUTUVCUYDNZUVQUUSBEUYEUVDUUKUVPUURUVCUVDUUKRUYDUVCUVDUUKUVCUVDNUUKUUHEUU
+  IXOXPXQXRUYEUVPUXDUUMQZUXDUUQQZRZGUUGPZUURUYEUVOUYHGUUGUYEUYFUVIUVNUYGUVCUYDU
+  YFUVIUVCUYDUYFWCZUXGUVEUVGWCZCTZUVIUYJUVHCTZUYLUYJUYFUYMUVCUYDUYFXSCUXDUULWAW
+  IUYJUVHUYKCUYJUVHUYKUYJUVHNZUXGUVEUVGUYNUULEUVFUVCUYDUYFUVHXTUYJUVEUVGYAZWHUY
+  JUVEUVGYBUYOYCXQYDYLUVIUXGUVHNZCTUYLUVHCEWEUYKUYPCUXGUVEUVGYEVTYFWRWSUVNUYGRU
+  YEUYGUVNUVJUVKUUPQZNZDTUVKUULQZUVMNZDTUYGUVNUYRUYTDUYRUVJUYSUVLNZNUYTUYQVUAUV
+  JUYQUYSUXTNVUAUVKUULUUOVPUXTUVLUYSUXTUYAUVLDVLUYBYGVRVSVRUYSUVJUVLVOYFVTDUXDU
+  UPWAUVMDUULWEWBWFUHYHYIUXDUUNQZUYGRZGSUXFUYHRZGSUURUYIVUCVUDGVUCUXFUYFNZUYGRV
+  UDVUBVUEUYGUXDUUGUUMVPYJUXFUYFUYGYKVSYMGUUNUUQYNUYHGUUGYOWBYPYQYRYSYTUUAUUCUU
+  DUUBUUEUUF $.
+  $}
+
+  ${
+    $d f i k l m n p r u w y z $.  $d f i k l m n p q v w y z $.
+    $( Express "every set is contained in a Grothendieck universe" in a short
+       form while avoiding complicated definitions.  (Contributed by Rohan
+       Ridenour, 8-Oct-2024.) $)
+    rr-grothshortbi $p |- ( A. x E. y e. Univ x e. y <->
+      A. x E. y ( x e. y /\ A. z e. y A. f e. ~P y E. w e. y
+        ( ~P z C_ ( y i^i w )
+          /\ ( z i^i U. f ) C_ U. ( f i^i ~P ~P w ) ) ) ) $=
```

Comment:
Do we have 
```
Univ = { y | A. z e. y A. f e. ~P y E. w e. y
        ( ~P z C_ ( y i^i w )
          /\ ( z i^i U. f ) C_ U. ( f i^i ~P ~P w ) ) }
```
? If true, it should probably be stated that way instead.

Reply:
(or even: `y e. Univ <-> ...`)

Reply:
The point of this theorem is to show that ~ rr-grothshort is equivalent to ~ gruex , and rewriting it in either of those ways would obscure that.

Reply:
Then, you can leave rr-grothshortbi, and prove it from the statement I propose above, namely
```
|- ( y e. Univ <-> A. z e. y A. f e. ~P y E. w e. y
        ( ~P z C_ ( y i^i w )
          /\ ( z i^i U. f ) C_ U. ( f i^i ~P ~P w ) ) )
```
which I think is worth stating.  But since that's in your mathbox, that's not a requirement.

Reply:
I added the first statement you posted as ~ dfuniv2 .